src/components/network/PoweredBy.vue

```diff
@@ -4,7 +4,8 @@
     class="powered-by-container"
     :class="{
       'powered-by-line': poweredByLine,
-      'hide-on-mobile': hideOnMobile
+      'hide-on-mobile': hideOnMobile,
+      'on-menu': isMenu
```

Comment:
I hope validators don't get annoyed that we are not displaying this on mobile 😅 

Should be OK though right

Reply:
oh yeah - i didn't even realize that.

Reply:
xD

Reply:
Well we can revert 😅  Maybe I was too quick